packages/houdini/src/runtime/cache/cache.ts

```diff
@@ -828,13 +828,34 @@ class CacheInternal {
 		// look at every field in the parentFields
 		for (const [
 			attributeName,
-			{ type, keyRaw, selection: fieldSelection, nullable, list, visible },
+			{ type, keyRaw, selection: fieldSelection, nullable, list, visible, directives },
 		] of Object.entries(targetSelection)) {
 			// skip masked fields when reading values
 			if (!visible && !ignoreMasking) {
 				continue
 			}
 
+			// some directives like @if and @include prevent the value from being in the
```

Comment:
Small mistake in the comment 🙂 
```suggestion
			// some directives like @skip and @include prevent the value from being in the
```

Reply:
Thanks!